src/tests/plugins/test_sendmail.py

```diff
@@ -94,5 +93,84 @@ def test_sendmail_one_message(logged_in_client, sendmail_url, event, order, expe
     url = sendmail_url + 'history/'
     response = logged_in_client.get(url)
 
-    assert response.status_code == expected
+    assert response.status_code == 200
     assert 'Test subject' in response.rendered_content
+
+
+@pytest.mark.django_db
+def test_sendmail_email_not_sent_to_order(logged_in_client, sendmail_url, event, order):
```

Comment:
Right! I will do that.
Hey, sorry I only addressed Raphael. I didn't know you were going to review. Thanks, by the way! :+1: